package.json

```diff
@@ -5,127 +5,137 @@
     "version": "0.0.1",
     "publisher": "mohitsuman",
     "repository": {
-        "type": "git",
```

Comment:
@sudhirverma 
1. For this specific issue, if `package.json` changes are not relevant, lets not include in the commit.
2. The spacing issues which you are having might be due to syntax defined in the editor you are using. The previous spacings used are *Spaces: 4* . Which editor are you using ?